test/from-draft3/to-draft4/disallow-has-atleast-one-object-in-its-array-initialize-allOf.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "title": "`disallow` is present and contains atleast one object in it",
+        "from": {
+            "disallow": [ "any", {}, "string" ]
```

Comment:
This test contradicts the one before. They both have the same `from` but different `to`?

Reply:
Okay, i'll remove `any`, because that will contradict